src/helpers/redact-strings-in-log-args.helper.ts

```diff
@@ -0,0 +1,31 @@
+import { get, set } from 'lodash';
+
+// This helper function is used to redact sensitive data in Error object strings.
+export const redactStringsInLogArgs = (
+  redactLogs: boolean,
+  redactPaths: string[],
+  redactStrings: { searchValue: string | RegExp; replaceValue: string }[],
+  args: any[],
+): any => {
+  if (!redactLogs) {
+    return args;
+  }
+  args.forEach((arg, index) => {
+    redactPaths.forEach((path) => {
+      const value: string = get(arg, path);
+      if (value) {
+        const safeValue = redactString(redactStrings, value);
+        set(args[index], path, safeValue);
+      }
+    });
+  });
+  return args;
+};
+
+const redactString = (redactStrings: { searchValue: string | RegExp; replaceValue: string }[], string: string): string => {
```

Comment:
Can a more meaningful name please be given to argument `string`, perhaps `message`?

Reply:
Thanks, changed to `logMessage`